defaults.yaml

```diff
@@ -0,0 +1,11 @@
+---
+dbUser: 'probodb'
+dbPassword: 'password'
+dbName: 'probodb'
+dbType: 'postgres'
+dbHost: 'localhost'
+eventGroup: 'proboDb'
+eventTopic: 'build_events'
+eventVersion: 1
+pluginDirectory: null
+knex: {}
```

Comment:
It is configurable now, you just have to start the server this way:
`NODE_ENV=development ./bin/probo-db`